### PR TITLE
Transfer wiki pages from git wiki to main repo

### PR DESCRIPTION
**Pull request by e-edling-ericsson**
### Description of the Change

The wiki-page functionality in github has some serious downsides:

Being in a separate repository compared to the main repository (e.g. eiffel-intelligence.wiki vs eiffel-intelligence) means that changes in functionality and changes in documentation will be two separate commits to two separate repositories. This makes the synchronization between actual functionality and documentation harder.
Changes in the wiki repository do not follow the normal pull-request review process, so other users cannot give any feedback before the commit is already made.
Since you cannot make pull-requests to the wiki repository, contributors need to have full write permissions in project to be able to make contributions.
### Possible Drawbacks

No default table of contents. We need to create our own.
### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: 
Erik Edling erik.edling@ericsson.com

### CODE REVIEW

**Comment by e-edling-ericsson** [comment]
If you found any outdated links, could you put them in the review as i have tried switching them out but obvsiously failed to find them all :) .

File: wiki/Subscription-Handling.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#### _Get Template_
+This button downloads a subscription template
+#### _Upload Subscriptions_
+This button help uploads a subscription by opening a file explorer
+#### _EI Backend Status_
+This button indicates, through its colour, whether a backend instance is connected with frontend or not. The green colour means backend is connected while red means no instance is connected.
+#### _Search_
+To search subscriptions with matching names in the database (displayed in the table).
+
+**_More Subscriptions related information can be found [here](https://github.com/eiffel-community/eiffel-intelligence/tree/master/wiki/Subscription-API.md) and [here](https://github.com/eiffel-community/eiffel-intelligence/tree/master/wiki/Subscriptions.md)_**
```
[the remark this answers is not in the document]
Comment (e-edling-ericsson) [reply wiki/Subscription-Handling.md]: They seem to work now that EI backend is merged.

[the remark this answers is not in the document]
Comment (J-Binieda-ericsson) [reply wiki/Subscription-Handling.md]: 👍

File: README.md [hunk 2]
```diff
@@ -34,3 +34,18 @@ To get involved, please see [Code of Conduct](./CODE_OF_CONDUCT.md) and [contrib
...
 
 Visit [Eiffel Community](https://eiffel-community.github.io) to get started and get involved.
+
+# Documentation
+
+1. [**GUI Overview**](./wiki/GUI-Overview.md)
+1. [**Subscription Handling**](./wiki/Subscription-Handling.md)
+    -  [**Add Subscription**](./wiki/Add-Subscription.md)
+1. [**Test Rules**](./wiki/Test-Rules.md)
+    - [**Rule testing mechanism via "Test Rules" GUI interface**](./wiki/Test-Rules.md#Rule-testing-mechanism-via-"Test-Rules"-GUI-interface)
```
Comment (roos-ericsson) [inline README.md]: Broken link. Remove " from url

File: README.md [hunk 3]
```diff
@@ -34,3 +34,18 @@ To get involved, please see [Code of Conduct](./CODE_OF_CONDUCT.md) and [contrib
...
+    - [**Introduction**](./wiki/Test-Rules.md#Introduction)
+    - [**Aggregated object**](./wiki/Test-Rules.md#Aggregated-object)
+    - [**Rule set up**](./wiki/Test-Rules.md#Rule-set-up)
+    - [**The most common operation you would do**](./wiki/Test-Rules.md#The-most-common-operation-you-would-do)
+1. [**Eiffel Intelligence Information**](./wiki/Eiffel-Intelligence-Information.md)
```
Comment (roos-ericsson) [inline README.md]: Remove this link and the following 2 links, and corresponding wiki pages, they do not contribute anything useful.

File: wiki/Eiffel-Intelligence-Information.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Eiffel Intelligence Information
```
Comment (roos-ericsson) [inline wiki/Eiffel-Intelligence-Information.md]: Remove this entire file. It does not add much useful information in my opinion.

File: wiki/Documentation-Links.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Documentation Links
```
Comment (roos-ericsson) [inline wiki/Documentation-Links.md]: Remove this entire file. It does not add much useful information in my opinion.

File: wiki/Administration-backend-instances.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# Administration Of Backend Instances
```
Comment (roos-ericsson) [inline wiki/Administration-backend-instances.md]: Remove this entire file. It does not add much useful information in my opinion.

**Pull request merged**